src/components/Network/index.js

```diff
@@ -323,6 +338,7 @@ class Network extends Component {
                   <ConstituentDatasets
                     key={constituentKey}
                     network={network}
+                    uuids={[uuids]}
```

Comment:
It is already a uuid, this will fail for users with the network scope i expect

Reply:
fixed